README.md

```diff
@@ -90,6 +90,11 @@ All properties mentioned below must be nested under the `style` prop. Examples o
 -   If you set the prop `useNativeAndroidPickerStyle` to false, the component will allow a few other style objects: `inputAndroidContainer`, `placeholder`, and `inputAndroid`
 -   Other styles that can be modified for Android are named `headlessAndroidContainer` and `viewContainer`
 
+#### Web-specific
+
+-   The component creates a <select> tag.
+-   The styling of this select tag can be modified using the prop `inputWeb`
```

Comment:
not a prop so much as a nested object within the `style` prop like the rest. "prop" is confusing here.

Reply:
Fair enough, I was having trouble thinking of a good name for it too, I can change it